ocpmodels/datasets/single_point_lmdb.py

```diff
@@ -68,6 +65,7 @@ def connect_db(self, lmdb_path=None):
             readonly=True,
             lock=False,
             readahead=False,
-            map_size=1099511627776 * 2,
+            meminit=False,
+            max_readers=1,
         )
         return env
```

Comment:
Add a `close()` method to the dataset class that closes the lmdb file. Training loop can call this method at the end of training.